code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Toxins.dm

```diff
@@ -780,4 +780,26 @@
 	..()
 	M.hallucination = max(M.hallucination, 50)
 	if(prob(10))
-		M.see_invisible = SEE_INVISIBLE_CULT
\ No newline at end of file
+		M.see_invisible = SEE_INVISIBLE_CULT
+
+/datum/reagent/toxin/trioxin
+	name = "Trioxin"
+	id = "trioxin"
+	description = "A synthetic compound of unknown origins, designated as a performance enhancing substance originally."
```

Comment:
*A synthetic compound of unknown origins, designated originally as a performance enhancing substance.